src/components/Modals/Send/views/Address.tsx

```diff
@@ -40,6 +40,10 @@ export const Address = () => {
   const { send } = useModal()
   const asset = useWatch<SendInput, SendFormFields.Asset>({ name: SendFormFields.Asset })
   const isYatFeatureEnabled = useFeatureFlag('Yat')
+
+  // reset address when asset changes
+  useEffect(() => setValue(SendFormFields.Input, ''), [asset, setValue])
```

Comment:
this was an existing bug - when switching assets it would not clear the address input or trigger validation, so we could potentially try and send an asset to an address on the wrong chain